shell/components/nav/TopLevelMenu.vue

```diff
@@ -998,7 +998,7 @@ export default {
     }
 
      &.menu-open {
-      width: 300px;
+      width: 315px;
```

Comment:
Increasing the width will have a side effect on how much TopLevelMenu covers the side bar,

before:
<img width="534" alt="image" src="https://github.com/user-attachments/assets/d1691da5-a057-4c6c-9fb6-a9c03aafe5a7">


after:
<img width="543" alt="image" src="https://github.com/user-attachments/assets/50ce5605-b5c8-4683-8ca5-6127c879b2d0">


I think it's fine, but getting an approval from someone in the design team would be nice.

Reply:
Good point, I'll check with them